src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/ReadHelper.cs

```diff
@@ -253,6 +256,7 @@ FROM entity_role_assignments_aggregate_history
     LIMIT 1
 ) earah ON true;")
             .AsNoTracking()
+            .WithQueryName()
```

Comment:
Just thinking if `.AnnotateMetricName()` or similar wouldn't be a better name. To make it clear it's for metrics purposes only.